File: src/ItsTheNumberEvenOrOdd.java
//Is the Number Even or Odd?
//https://edabit.com/challenge/umRwYETG7ogToZhSi

public class ItsTheNumberEvenOrOdd {
    public static void main(String[] args){
        System.out.println("Is the number even or odd (3) -> " + isEvenOrOdd(3));
        System.out.println("Is the number even or odd (126) -> " + isEvenOrOdd(126));
        System.out.println("Is the number even or odd (19) -> " + isEvenOrOdd(19));
        System.out.println("Is the number even or odd (1) -> " + isEvenOrOdd(1));
    }
    public static String isEvenOrOdd(int num) {
        if(num % 2 == 0) {
            return "even";
        }
        else{
            return "odd";
        }
    }
}

File: src/itsTheNumberEvenOrOdd.js
// Is the Number Even or Odd?
// https://edabit.com/challenge/kuzB5CMXiKDEYKXAP

function isEvenOrOdd(num) {
  if (num % 2 == 0) {
    return "even";
  } else {
    return "odd";
  }
}

console.log("Is the number even or odd (3) -> " + isEvenOrOdd(3));
console.log("Is the number even or odd (126) -> " + isEvenOrOdd(126));
console.log("Is the number even or odd (19) -> " + isEvenOrOdd(19));
console.log("Is the number even or odd (1) -> " + isEvenOrOdd(1));
